src/database/models/user.js

```diff
@@ -1,14 +1,13 @@
-import { Model } from 'sequelize';
+const { Model } = require('sequelize');
 
-export default (sequelize, DataTypes) => {
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Change this back to ES6, please. Refer to previous comment on database/models/index.js